app/services/travel_to_work_area_and_london_borough_service.rb

```diff
@@ -1,20 +1,60 @@
 class TravelToWorkAreaAndLondonBoroughService
-  def self.add_travel_to_work_area_and_london_borough(site:)
-    url = URI("#{Settings.mapit_url}/point/4326/#{site.longitude},#{site.latitude}?type=TTW,LBO&api_key=#{Settings.mapit_api_key}")
-    response = Net::HTTP.get(url)
-    json = JSON.parse(response)
-
-    ttw_area = json.dig(json.keys.first, "name")
-
-    london_borough = if ttw_area == "London"
-                       json
-                       .dig(json.keys.second, "name")
-                       .gsub(/ Borough Council| City Council| Corporation/, "")
-                     end
-
-    site.update_column("travel_to_work_area", ttw_area)
-    site.update_column("london_borough", london_borough)
-  rescue StandardError
-    false
+  include ServicePattern
+
+  def initialize(site:)
+    @site = site
+  end
+
+  def call
+    ttw_area = get(:travel_to_work_area)
+
+    if ttw_area == "London"
+      london_borough = get(:london_borough)
+
+      if london_borough
+        site.travel_to_work_area = "London"
+        site.london_borough = london_borough
+        site.save!(validate: false)
+      end
```

Comment:
If area is "London" but it's not a london_borough, `call` returns `nil`, whereas if no `ttw_area` is provided `false` is returned. Correct? Does it matter?

Reply:
Ah good spot - should be returning false - will change 👍